### ISSUE TITLE
Can signtypeddata be supported?

### ISSUE THREAD

**Issue by xhzth70911** [Boilertalk/Web3.swift#123]
Signtypeddata doesn't have this method. Can it be implemented like web3.js? thank you

**Comment by koraykoska** [comment]
@xhzth70911 No, this is not supported. You are open to implementing it and opening a PR.

**Comment by danijelTxFusion** [comment]
What the current status of this issue. Has someone implemented on some forked repo?
